### PR TITLE
Migrate redesign

### PR DESCRIPTION
**Pull request by amitaibu**
Redesign homepage cards, with the new migrated content.

image

### CODE REVIEW

**Comment by amitaibu** [comment]
Oh, weird - it opened it under your Repo :)

File: web/themes/custom/server_theme/src/scss/components/typography.scss [hunk 1]
```diff
@@ -1,3 +1,11 @@
 h1, h2, h3, h4 {
   @apply font-headers leading-none text-gray-700;
 }
+
+.fluid-container-wide {
+  @apply max-w-7xl mx-auto px-4 sm:px-6 lg:px-8;
```
Comment (bboro) [inline web/themes/custom/server_theme/src/scss/components/typography.scss:6]:
```suggestion
  @apply max-w-7xl mx-auto px-4;
  
  @screen sm {
    @apply px-6;
  }
  
  @screen lg {
    @apply px-8;
  }
```

Do the responsive classes work with @apply? From what I can remember with Tailwind 1 it doesn't work. Maybe it works now?

File: web/themes/custom/server_theme/src/scss/components/typography.scss [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+}
+
+.fluid-container-narrow {
+  @apply max-w-6xl mx-auto px-4 sm:px-6 lg:px-8;
```
Comment (bboro) [inline web/themes/custom/server_theme/src/scss/components/typography.scss:10]: Also here

**Review by bboro — COMMENTED** [review]
Just spotted one thing, not sure if valid or not :)

File: web/themes/custom/server_theme/src/scss/components/typography.scss (hunk 1 above)
Comment (amitaibu) [reply web/themes/custom/server_theme/src/scss/components/typography.scss:6]: Yeah, it's works with TW2

Comment (bboro) [reply web/themes/custom/server_theme/src/scss/components/typography.scss:6]: Neat!

**Comment by bboro** [comment]
Assuming this is finished. I'm merging.

**Pull request merged**